usage-examples/index.html

```diff
@@ -0,0 +1,96 @@
+<head>
```

Comment:
This is a resulting file, isn't it?
If so it shouldn't be included in the commit but rather `~.html` be added to the `.gitignore` file.